### PR TITLE
add popup

### PR DESCRIPTION
**Pull request by Natalka2019**
### TML-CSS-Popup

[Demo](https://natalka2019.github.io/kottans-popup/) |
[Code base](https://github.com/Natalka2019/kottans-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Natalka2019** [comment]
I have completed self-checks and believe that my code is in conformity with recommendations

File: submissions/nrushchak/popup/index.css [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+}
+
+.arrow {
+  width: 30px;
+  height: 30px;
+  top: 75px;
+  right: 120px;
+  transform: rotate(45deg);
+  background-color: white;
+  z-index: 1000;
```
Comment (A-Ostrovnyy) [inline submissions/nrushchak/popup/index.css:135]: Avoid such big numbers.

File: submissions/nrushchak/popup/index.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+          <li class="header__nav__list__item">
+            <a class="header__nav__list__item__link title" href="#">Mail</a>
+          </li>
+          <li class="header__nav__list__item">
+            <a class="header__nav__list__item__link title" href="#">Images</a>
+          </li>
+          <li class="header__nav__list__item">
+            <input class="nav__checkbox__input" type="checkbox" id="popup-checkbox" name="popup-checkbox" />
+            <label for="popup-checkbox" class="nav__checkbox__label">
+              <img class="nav__checkbox__image" src="popup-icons/popup-button.png" alt="popup button" />
```
Comment (A-Ostrovnyy) [inline submissions/nrushchak/popup/index.html:26]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

File: submissions/nrushchak/popup/index.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            </label>
+
+            <div class="popup">
```
Comment (A-Ostrovnyy) [inline submissions/nrushchak/popup/index.html:29]: Fix the position of the popup according to the button. Also, fix `z-index` of popup and footer
image

File: submissions/nrushchak/popup/index.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+          <li class="header__nav__list__item">
+            <a class="header__nav__list__item__link" href="#">
+              <img class="avatar" src="https://cdn4.iconfinder.com/data/icons/avatar-vol-1-3/512/9-256.png"
+                alt="user's avatar" />
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main"></main>
```
Comment (A-Ostrovnyy) [inline submissions/nrushchak/popup/index.html:172]: Avoid unused lines of code in your work.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good Work!

**Pull request merged**